app/src/main/java/dev/leonlatsch/photok/settings/ui/AboutScreen.kt

```diff
@@ -161,6 +165,22 @@ fun AboutScreen(
                     Text(BuildConfig.VERSION_NAME)
                 }
 
+                if (vaultStats != null) {
```

Comment:
This UI is not idea. Maybe you can put the thats more in the bottom.

I would also like a monospaced font for this and for it to be in a card or something. If you are not that into UI stuff, I can merge it like this later and make a UI overhaul myself.